public/js/controllers/StatsController.js

```diff
@@ -610,21 +644,36 @@ angular.module('BlocksApp').controller('StatsController', function($stateParams,
                 .attr("x", 9)
                 .attr("dy", ".35em");
 
+            svg.call(tip);
+
             svg.append("rect")
                 .attr("class", "overlay")
                 .attr("width", width)
                 .attr("height", height)
-                .on("mouseover", function() { focus.style("display", null); })
-                .on("mouseout", function() { focus.style("display", "none"); })
+                .on("mouseover", function(d) {

```

Comment:
same comment about passing the `d` parameter. I notice in this example here: http://bl.ocks.org/d3noob/a22c42db65eb00d4e369 That they pass the `d` parameter, but they're using it inside the function. For example, 
```
.on("mouseover", function(d) {		
            div.transition()		
                .duration(200)		
                .style("opacity", .9);		
            div	.html(formatTime(d.date) + "<br/>"  + d.close)	
                .style("left", (d3.event.pageX) + "px")		
                .style("top", (d3.event.pageY - 28) + "px");	
            })		
```
Here, the `d` is being referenced inside the function in `d.date` and `d.close`